app/src/main/java/com/osfans/trime/Composition.java

```diff
@@ -331,14 +328,14 @@ private int calc_start_num(int min_length, int min_check) {
 
     if(j<0)
       j=0;
+
     for(;j<max_entries && j<candidates.length;j++){
```

Comment:
init j in for loop?
```
    for(j = 0; j<max_entries && j<candidates.length; j++){
```

Reply:
> init j in for loop?
> ```
>     for(j = 0; j<max_entries && j<candidates.length; j++){
> ```

j doesn't need init again, cause of the first loop has set a value to j

Reply:
I think better naming conversion needed here